exporter/signalfxexporter/internal/translation/converter.go

```diff
@@ -88,12 +88,12 @@ func (c *MetricsConverter) MetricsToSignalFxV2(md pmetric.Metrics) []*sfxpb.Data
 			ilm := rm.ScopeMetrics().At(j)
 			for k := 0; k < ilm.Metrics().Len(); k++ {
 				dps := c.translator.FromMetric(ilm.Metrics().At(k), extraDimensions)
-				dps = c.translateAndFilter(dps)
 				sfxDataPoints = append(sfxDataPoints, dps...)
 			}
 		}
 	}
 
+	sfxDataPoints = c.translateAndFilter(sfxDataPoints)
```

Comment:
Given that a lot of unrelated resource metrics can be merged together by the batch processor, I think we should move it to the resource scope, right after `for j := 0; j < rm.ScopeMetrics().Len(); j++ {}`

Reply:
Will do, thanks for pointing this out.